src/theme.ts

```diff
@@ -43,7 +53,10 @@ export default createTheme({
     MuiContainer: {
       root: {
         width: '950px',
-        maxWidth: `calc(100vw - ${mobileBrandSidebarWidth}px)`,
+        maxWidth: `calc(100vw - ${tabletBrandSidebarWidth}px)`,
+        [defaultTheme.breakpoints.down(BREAKPOINTS.values.md)]: {
+          maxWidth: '100vw',
```

Comment:
Here we could consider 
```
width: '100vw',
maxWidth: '550px' // or something 
```
to keep things from being too wide at a mobile width of, say, 767.